packages/shared/src/graphql/users.ts

```diff
@@ -76,3 +76,12 @@ export const USER_READING_HISTORY_QUERY = gql`
     }
   }
 `;
+
+export const UPVOTER_FRAGMENT = gql`
```

Comment:
I thought that you can include all the fields of the upvote request in the fragment. Including the pageInfo and edge

Reply:
Unfortunately, the `Connection` type of the `schema` of Upvote for `posts` is different from the `Connection` type of the `schema` of Upvote for `comments`.